app/helpers/login_helper.rb

```diff
@@ -0,0 +1,9 @@
+module LoginHelper
+  def mount_login
+    props = { token: form_authenticity_token,
```

Comment:
As discussed offline, can you please move properties generation into a separate method, so it wil be possible to override the props without redefining component rendering?

``` ruby
def login_props
{
  token: form_authenticity_token,
  version: SETTINGS[:version].version,
  caption: Setting[:login_text],
  alert: flash_inline
}

def mount_login
  render('common/login', props: login_props)
end
```